src/dkg/pv.rs

```diff
@@ -91,17 +93,13 @@ where
     /// Call `finish_announce` once the Announcement phase is complete
     /// Partitions the share domain among the announced participants
     /// and begins the sharing phase of the DKG
+    /// TODO: Since announce is no longer a phase, do we still need this?
     pub fn finish_announce(&mut self) -> Result<()> {
```

Comment:
Maybe let's pass the announcements in here as a parameter. And maybe we should just rename announcements to stakes or something since we are announcing anything anymore, just querying the tendermint node.

Reply:
Also, maybe we should rename this function to `create_weight_shares` or something.